contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -4339,34 +4351,28 @@ pltsql_call_handler(PG_FUNCTION_ARGS)
 		PG_CATCH();
 		{
 			set_procid(prev_procid);
-			/* Decrement use-count, restore cur_estate, and propagate error */
 			pltsql_trigger_depth = save_pltsql_trigger_depth;
-			func->use_count--;
-			func->cur_estate = save_cur_estate;
-			pltsql_remove_current_query_env();
-			pltsql_revert_guc(save_nestlevel);
-			pltsql_revert_last_scope_identity(scope_level);
-			terminate_batch(true /* send_error */ , false /* compile_error */ );
-			sql_dialect = saved_dialect;
-			return retval;
+			
+			send_error = true;
 		}
 		PG_END_TRY();
+		
+		/* Decrement use-count, restore cur_estate, and propagate error */
+		func->use_count--;
+
+		func->cur_estate = save_cur_estate;
+
+		pltsql_remove_current_query_env();
+		pltsql_revert_guc(save_nestlevel);
+		pltsql_revert_last_scope_identity(scope_level);
 	}
 	PG_FINALLY();
 	{
 		sql_dialect = saved_dialect;
 	}
 	PG_END_TRY();
 
-	func->use_count--;
-
-	func->cur_estate = save_cur_estate;
-
-	pltsql_remove_current_query_env();
-	pltsql_revert_guc(save_nestlevel);
-	pltsql_revert_last_scope_identity(scope_level);
-
-	terminate_batch(false /* send_error */ , false /* compile_error */ );
```

Comment:
these changes fix static code analyzer warnings.
Uninitialized variable error is thrown for func, save_nestlevel, save_nestlevel 

for example 
contrib/babelfishpg_tsql/src/pl_handler.c:4374:2: warning: Uninitialized variable: func [uninitvar]
 func->cur_estate = save_cur_estate;
 ^
contrib/babelfishpg_tsql/src/pl_handler.c:4284:64: note: Assuming condition is false
 if ((rc = SPI_connect_ext(nonatomic ? SPI_OPT_NONATOMIC : 0)) != SPI_OK_CONNECT)
                                                               ^
contrib/babelfishpg_tsql/src/pl_handler.c:4293:2: note: Assuming condition is false
 PG_TRY();
 ^